builder/azure/common/config_test.go

```diff
@@ -0,0 +1,44 @@
+package common_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/hashicorp/packer-plugin-azure/builder/azure/common"
+)
+
+func TestSkipCreateImage(t *testing.T) {
```

Comment:
I think we should add a test case here where we pass in false for SkipCreateImage, but this is good for now.